README.md

```diff
@@ -14,3 +14,14 @@ Name: Saunov Abish
 **Link to File**: https://github.com/JaydevSR/IISERB-s_favorite_algorithms/blob/main/algorithms/example.txt
 
 This is the description of example algorithm. Add your algorithms **below** (Please maintain alphabetical order)
+
+
+### Bellman-Ford Algorithm
+
+Name : Kumar Katyayan Jaiswal
+
+**Link to File**: https://github.com/JaydevSR/IISERB-s_favorite_algorithms/blob/main/algorithms/bellman_ford.py
+
+Implementation of the Bellman-Ford algorithm in python. 
```

Comment:
After this you can also add execution instructions if any (in your case it is just running `python filename.py`).

**TIP**: In a larger project a library will most probably export some functions for direct use of the users (for example scipy exports scipy.integrate etc.). In such case the documentation will consist of something like:

```
function(arg1, arg2, ...)

Performs/Calculates something and something.

More explaination ...

Examples:
    ```
    an example
    ```
```

This is basically all there is to documenting code for others.